planner/solver.go

```diff
@@ -262,8 +263,8 @@ func FindBestPlanningSolutions(ctx context.Context, placeClusters [][]matching.P
 		topSolutionsCount = TopSolutionsCountDefault
 	}
 
-  priorityQueue := &MinPriorityQueue[Vertex]{}
-  includedPlaces := make(map[string]bool)
+	priorityQueue := &MinPriorityQueue[Vertex]{}
+	includedPlaces := make(map[string]int8)
```

Comment:
Can you please add in a comment the rationale for using int8 values?

Reply:
Done, added in 2137604.